docs/clientdev/interfaces/index.rst

```diff
@@ -9,9 +9,9 @@ including storing stacktraces, HTTP request information, and other
 metadata.
 
 For the most part interfaces are an evolving part of Sentry.  Like with
-attributes, clients are expected to assume more can appear at any point in
+attributes, SDK are expected to assume more can appear at any point in
```

Comment:
seems there are quite a few plurality mismatches, gonna stop calling them each out (but not *all* are mismatched)